Cloudp 126274 (#1342)
mongodb/mongodb-atlas-cli: docs/atlascli/command/atlas-clusters-upgrade.txt

.. _atlas-clusters-upgrade:

======================
atlas clusters upgrade
======================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

> Upgrade a MongoDB Atlas cluster.

Reviewer: ```suggestion
Upgrade a MongoDB Atlas cluster's tier, disk size, and/or MongoDB version.
```

Syntax
------

.. code-block::

   atlas clusters upgrade [clusterName] [options]

Arguments
---------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - clusterName
     - string
     - false

>      - Name of the cluster to upgrade

Reviewer: ```suggestion
     - Name of the cluster to upgrade.
```

Options
-------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - --diskSizeGB
     - float
     - false
     - Capacity, in gigabytes, of the host's root volume.
   * - -f, --file
     - string
     - false
     - File name to use, optional file with a json cluster configuration.
   * - -h, --help
     - 
     - false
     - help for upgrade
   * - --mdbVersion
     - string
     - false

>      - MongoDB version of the cluster to deploy.

Reviewer: We should take out the "to deploy" since this no longer on applies to deploying new ones
```suggestion
     - MongoDB version of the cluster.
```

   * - -o, --output
     - string
     - false
     - Output format. Valid values are json, json-path, go-template, or go-template-file.
   * - --projectId
     - string
     - false
     - Project ID to use. Overrides the settings in the configuration file or environment variable.
   * - --tier
     - string
     - false
     - Tier for each data-bearing server in the cluster.

Inherited Options
-----------------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - -P, --profile
     - string
     - false
     - Profile to use from your configuration file.

Examples
--------

.. code-block::

>    Upgrade tier, disk size and MongoDB version for a cluster:

Reviewer: ```suggestion
   The following example upgrades the tier, disk size, and MongoDB version for a cluster:
```
